main.js

```diff
@@ -107,23 +107,26 @@ function findIdeaIndex(e) {
 
 function handleFocusOut(e) {
   if (e.target.className === 'card-title' || e.target.className === 'card-body') {
-    console.log("event:", event);
     var newTitle = e.target.closest('.card-content').querySelector('.card-title').innerText;
     var newBody = e.target.closest('.card-content').querySelector('.card-body').innerText;
     var index = findIdeaIndex(e);
 
     ideasArray[index].updateIdea(newTitle, newBody);
     ideasArray[index].saveToStorage(ideasArray);
-    console.log("index:", index);
-    // console.log(title.innerText);
-    // console.log(body.innerText);
   }
 }
 
 function toggleStar(e) {
-  if (e.target.id === 'btn-star') {
-    var on = document.querySelector('.star-active').classList.remove('hidden');
-    var off = document.querySelector('.star').classList.add('hidden');
+  if(event.target.classList.contains('img-star')) {
```

Comment:
Make sure you parameter and event object syntax match. Do we want to use e or event? Looks like we are using e in other places in our code.